azure-devops/azext_devops/dev/team/service_endpoint.py

```diff
@@ -129,7 +130,8 @@ def create_service_endpoint(service_endpoint_type, authorization_scheme, name,
                 'subscriptionId': azure_rm_subscription_id,
                 'subscriptionName': azure_rm_subscription_name,
                 'environment': 'AzureCloud',
-                'creationMode': 'Manual'
+                'creationMode': 'Manual',
+                'GeneratedBy': 'AzureDevopsExtensionFlow'
```

Comment:
maybe we can put it just above line 137
check if data exists, if not then add data and then add GeneratedBy to the data  (so that every time we add support for new type then we don't have to remember it) and this is common across SE types so that is other reason
`return client.create_service_endpoint(service_endpoint_to_create, project)`